### PR TITLE
1.16-stable jobs

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

**Comment by GregSutcliffe** [comment]
LGTM - @ehelms / @mmoll ?

File: puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_proxy_1_16_stable.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    triggers:
+      - scm_fifteen_minutes
+      - github
+      - schedule_failed_builds
+    axes:
+      - axis:
+          type: user-defined
+          name: ruby
+          values:
+          - 2.1
```
Comment (mmoll) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_proxy_1_16_stable.yaml]: the proxy still needs to support Ruby 2.0.0 (e.g. on EL 7)

**Comment by mmoll** [comment]
@dLobatog did you forget to push? ;)

**Comment by mmoll** [comment]
merged, gracias @dLobatog!

**Pull request merged**